WinFormsUI/MainForm.cs

```diff
@@ -88,38 +88,70 @@ private void RefreshList()
 
 		private void RefreshChart()
 		{
-			chartSeries.Series.Clear();
+			chartSeriesSmoothed.Series.Clear();
+			chartSeriesCumulative.Series.Clear();
 
 			var selectedCategories = clbCategories.CheckedItems
 			                                      .Cast<object>()
 			                                      .Select(clbCategories.GetItemText)
 			                                      .ToList();
-			var seriesToRemove = chartSeries.Series.Where(s => selectedCategories.All(c => c != s.Name)).ToList();
 
-			foreach (var s in seriesToRemove)
+			var smoothedSeriesToRemove = chartSeriesSmoothed.Series.Where(s => selectedCategories.All(c => c != s.Name));
+			var cumulativeSeriesToRemove = chartSeriesCumulative.Series.Where(s => selectedCategories.All(c => c != s.Name));
+
+			foreach (var s in smoothedSeriesToRemove)
 			{
-				chartSeries.Series.Remove(s);
+				chartSeriesSmoothed.Series.Remove(s);
+			}
+
+			foreach (var s in cumulativeSeriesToRemove)
+			{
+				chartSeriesCumulative.Series.Remove(s);
 			}
 
 			foreach (var c in selectedCategories)
 			{
-				if (chartSeries.Series.FindByName(c) is null)
+				if (chartSeriesSmoothed.Series.FindByName(c) is null)
 				{
 					var newSeries = new Series
 					{
 						Name = c,
 						ChartType = SeriesChartType.Line
 					};
-					chartSeries.Series.Add(newSeries);
+					chartSeriesSmoothed.Series.Add(newSeries);
 				}
 
-				var series = chartSeries.Series.FindByName(c);
-				series.Points.Clear();
-				var timeSeries = State.GetTimeSeries(c, 0.99);
+				if (chartSeriesCumulative.Series.FindByName(c) is null)
+				{
+					var newSeries = new Series
+					{
+						Name = c,
+						ChartType = SeriesChartType.Line
+					};
+					chartSeriesCumulative.Series.Add(newSeries);
+				}
+
+				var smoothedSeries = chartSeriesSmoothed.Series.FindByName(c);
+				smoothedSeries.Points.Clear();
+
+				var cumulativeSeries = chartSeriesCumulative.Series.FindByName(c);
+				cumulativeSeries.Points.Clear();
+
+				var smoothingRatio = 0.99;
+				var increment = 100;
+				var capacity = 10000;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'const' modifier to 'capacity'.](https://app.codacy.com/manual/siriak/MoneyManager/pullRequest?prid=4992784)